.travis.yml

```diff
@@ -33,6 +33,7 @@ env:
     - TARGET=test
     - UPSTREAM_URL=https://github.com/sonata-project/SonataPageBundle.git
     - PHPUNIT_VERSION=7
+    - COMPOSER_MEMORY_LIMIT=-1
```

Comment:
This file is managed through dev-kit, which means your changes will disappear on next dev-kit deploy, you have to do something else. Besides, it's weird that Composer needs more memory and time to solve what looks like an easier problem…

Reply:
I removed this line and during lowest composer process there are : 

```
PHP Fatal error:  Allowed memory size of 3221225472 bytes exhausted (tried to allocate 8192 bytes) in phar:///home/travis/.phpenv/versions/7.2.15/bin/composer/src/Composer/DependencyResolver/GenericRule.php on line 36
```

Maybe just add `COMPOSER_MEMORY_LIMIT=-1` on this process ?

Reply:
Oh but it only happens for the `prefer-lowest` job apparently… that means that bumping some other dependencies to get that job closer to what the other jobs do could help (I know it can, I have fixed similar issues like this in the past).

Reply:
Ok, if I remove support of Symfony 2.8 and Twig 1.34, it's ok for you ?

Reply:
The issue of prefer-lowest job is resolved, I think it's a good choice, what do you think @greg0ire ?

Reply:
It looks ok, you can even use sf 3.4, versions below are not supported

Reply:
I just authorize sf ^3.4 and ^4.0